### PR TITLE
Import/Export/Share backends in settings

### PR DESCRIPTION
**Pull request by grubelic**
**Description**
Added option to import, export and share backends as JSON.

**Issues to solve**
#118

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
+      <module fileurl="file://$PROJECT_DIR$/Sensorics.iml" filepath="$PROJECT_DIR$/Sensorics.iml" />
```
Comment (borna-mdn) [inline .idea/modules.xml]: Remove duplicated line

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -18,7 +18,7 @@
     <uses-permission android:name="android.permission.BLUETOOTH_ADMIN" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```
Comment (borna-mdn) [inline app/src/main/AndroidManifest.xml]: Why did you remove READ_EXTERNAL_STORAGE?

Comment (grubelic) [reply app/src/main/AndroidManifest.xml]: Because granting WRITE_EXTERNAL_STORAGE permission implicitly grants permission to read the external storage as well.

Source: (https://developer.android.com/training/data-storage/files#ExternalStoragePermissions)

File: .idea/modules.xml (hunk 1 above)
Comment (Nicba1010) [reply .idea/modules.xml]: Fixed
